frontend/ui/impl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/RediscoverSuggester.java

```diff
@@ -71,4 +80,32 @@ private long daysDiffTillNow(long lastModifiedTimeMillis) {
     val millisDiff = currentTimeMillis - lastModifiedTimeMillis;
     return millisDiff / (24 * 60 * 60 * 1000);
   }
+
+  public void enqueueChecksThenMaybePerform() {
```

Comment:
nit: I do not like this name xd "maybe" sounds a bit random XD `checkAndPerformIfApplicable` maybe?

Reply:
True 😆